ros/src/waypoint_updater/srv/NextWaypoint.srv

```diff
@@ -1,4 +1,4 @@
 geometry_msgs/PoseStamped pose
-styx_msgs/Lane waypoints
+styx_msgs/Waypoint[] waypoints
```

Comment:
I think you should keep the Lane here,  this will be compatible with autoware.

Reply:
I didn't change this, if you update your master with upstream this change won't be here